ci/infra/vmware/terraform.tfvars.example

```diff
@@ -65,3 +65,7 @@ authorized_keys = []
 
 # IMPORTANT: Replace these ntp servers with ones from your infrastructure
 ntp_servers = ["0.novell.pool.ntp.org", "1.novell.pool.ntp.org", "2.novell.pool.ntp.org", "3.novell.pool.ntp.org"]
+
+# Setup for vSphere cloud provider
+setup_cloud_provider = false
+
```

Comment:
Sorry, I don't know much about this detail.
But what's the pros/cons not to default enable `setup_cloud_provider`?

Reply:
I've left the default disabled here because skuba currently support vmware provision but `--cloud-provider` is not a mandatory option. IMO this is ok for now so it left user with no surprised unless they are intentionally going for VCP cluster. But this just my 2 cents.

Reply:
Good ask and good answer :)